### DISCUSSION TITLE
Research Assistants

### DISCUSSION THREAD

**Discussion by mdlmarkham** [open-webui/open-webui#5133] [Ideas]
I've come across two projects that look like they could be very powerful in conjunction with OpenWebUI:

- [STORM](https://storm.genie.stanford.edu/) is a really interesting tool that uses a group of agents to build an article
- [gpt-researcher](https://github.com/assafelovic/gpt-researcher) seems similar to STORM, but a little more flexible

Both of these solutions have a very simple workflow... the user inputs a topic, and they go off, spin up a group of agents, research the topic and build an artifact. While this is very nifty, it isn't really a finished product. What if that document were then placed back into the context of a chat where it could be used as input for a model... or further refined through discussion... and finally added to a document library and used as reference?

**Comment by TomLucidor** [comment]
Seconding this here https://github.com/open-webui/open-webui/discussions/9321
